repo/packages/B/beta-kubernetes/8/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "5.0",
+  "minDcosReleaseVersion": "1.12",
+  "upgradesFrom": [
+    "*"
```

Comment:
🤔 I wonder if this can be a problem if we don't remove existing `beta-kubernetes` versions... Thoughts @MatApple?